python/ray/reporter.py

```diff
@@ -149,7 +150,11 @@ def get_workers():
         ]
 
     def get_load_avg(self):
-        load = os.getloadavg()
+        if sys.platform == "win32":
```

Comment:
Can we just use `psutil.cpu_percent()` for both cases now that `psutil` is bundled with ray?

Reply:
I'm not sure. `getloadavg` returns samples from the last 1, 5, and 15 minutes, but `cpu_percent` doesn't do that. I haven't checked why callers need the 3 samples, so I thought I'd avoid changing the existing behavior on Linux, but if you think we should change them I certainly can.

Reply:
I see, let's not change the existing behavior on linux then